otx/core/data/adapter/base_dataset_adapter.py

```diff
@@ -250,5 +250,20 @@ def _get_polygon_entity(self, annotation: DatumaroAnnotation, width: int, height
                     for i in range(0, len(annotation.points), 2)
                 ]
             ),
-            labels=[ScoredLabel(label=self.label_entities[annotation.label - 1])],
+            labels=[ScoredLabel(label=self.label_entities[annotation.label])],
         )
+
+    def remove_unused_label_entities(self, used_labels: List):
+        """Remove unused label from label entities.
+        
+        Because label entities will be used to make Label Schema,
+        If there is unused label in Label Schema, it will hurts the model performance.
+        So, remove the unused label from label entities.
+
+        Args:
+            used_labels (List): list for index of used label
+        """
+        clean_label_entities = []
+        for used_label in used_labels:
+            clean_label_entities.append(self.label_entities[used_label])
+        self.label_entities = clean_label_entities
```

Comment:
This seems to give a no newline error. Does the pre-commit pass?

Reply:
I missed the pre-commit test ! However, after the pre-commit, no big differences..
![image](https://user-images.githubusercontent.com/80735344/214513163-a32d1682-fb7f-4aa9-9b72-57307a1187b6.png)